src/cpp/rtps/messages/MessageReceiver.cpp

```diff
@@ -841,6 +841,8 @@ bool MessageReceiver::proc_Submsg_Data(
             {
                 EPROSIMA_LOG_WARNING(RTPS_MSG_IN, IDSTRING "Serialized Payload value invalid or larger than maximum allowed size"
                         "(" << payload_size << "/" << (msg->length - msg->pos) << ")");
+                ch.serializedPayload.data = nullptr;
+                ch.inline_qos.data = nullptr;
```

Comment:
Shouldn't we document this in the doxydoc?

Reply:
I don't understand this comment. `ch` is a local variable. What should we document in the doxydoc?